src/components/Tile.tsx

```diff
@@ -1,9 +1,11 @@
-import { FC, useEffect, useRef } from 'react';
+import { FC } from 'react';
 import './Tile.scss';
 
 type TileProps = {
   value: number;
   position: [number, number];
+  isMerged?: boolean;
+  isNew?: boolean;
```

Comment:
I think this variable can be removed. All the tiles are considered new if not merged.